bclconvert.wdl

```diff
@@ -211,16 +187,69 @@ task bclconvert {
   }
   
   output { 
-     File reports = "~{runName}.reports.gz"
      FastqCollection fastqs = read_json("outputs.json")
   }
-  
-
 }
 
- 
+# =================================
+# Run bclconvert on DRAGEN hardware
+# =================================
+task runBclconvertDragen {
+  input {
+    String runFolder
+    String runName = basename(runFolder)
+    File sampleSheet
+    Boolean firstTileOnly = false
+    Boolean noLaneSplitting = false
+    Boolean onlyMatchedReads = true
+    String fastqCompression = 'gzip'
+    String bclconvertScript = "/.mounts/labs/gsi/modulator/sw/Ubuntu20.04/bclconvert-scripts-1.0/bin/runBclconvert.py"
```

Comment:
I see you have `runBclconvertDragen.modules` defined in the readme, but not present here. I imagine you found that modules are not support on the dragen node, as it runs Oracle Linux and our modules are built for Ubuntu 20.04.

I think this is okay for now, but can you bring this up in the workflow meeting? We should get required modules built for Oracle Linux before this workflow goes into production.